src/VivViewerOverview.js

```diff
@@ -0,0 +1,149 @@
+import { OrthographicView } from '@deck.gl/core';
+import { OverviewLayer } from './layers';
+
+export default class VivViewerOverview {
+  constructor({
+    viewWidth,
+    viewHeight,
+    loader,
+    offset = 25,
+    overviewLocation = 'bottom-right',
+    id = 'overview',
+    boundingBoxColor = [255, 0, 0],
+    boundingBoxOutlineWidth = 50,
+    viewportOutlineColor = [255, 192, 204],
+    viewportOutlineWidth = 400,
+    overviewScale = 1
+  }) {
+    const { numLevels } = loader;
+    const rasterSize = loader.getRasterSize({
+      z: 0
+    });
+    this.offset = offset;
+    this.viewWidth = viewWidth;
+    this.viewHeight = viewHeight;
+    this.id = id;
+    this.overviewScale = overviewScale;
+    this._numLevels = numLevels;
+    // These are the pixel-space height and width
+    /* eslint-disable no-bitwise */
+    this.height = (rasterSize.height * overviewScale) >> (numLevels - 1);
+    this.width = (rasterSize.width * overviewScale) >> (numLevels - 1);
+    this.imageHeight = rasterSize.height;
+    this.imageWidth = rasterSize.width;
```

Comment:
This still doesn't have the effect that I would expect. The size of the overview layer depends on the size of the image and the number of levels in the image. For example, if you change `const MINZOOM = -7` in `App.js`, the overview is much much larger. This would be very odd if using `VivViewer` to look at different image pyramids of different sizes.

I was hoping that the `scale` config would explicitly specify how large the picture-in-picture is relative to the `viewWidth` or `viewHeight`. If `scale === 0.5` than the picture-in-picture would be 1/2 the size of the view port. Since the viewport is likely to be a different aspect ratio compared to the underlying image, perhaps `scaleWidth` would be better. If `scaleWidth === 0.5` then then width of the picture in picture image would be 1/2 the `viewWidth`.

Reply:
I'm actually not sure what the behavior of `overviewScale` is currently. I would expect an `overviewScale` of `0.7` to fill nearly the entire detail image.

Reply:
Ah, ok, I misunderstood what you were saying with scale.  I understand now - I thought you were saying to scale the viewport by what the dimensions of the detail image are (which I best approximated) not the viewport relative to the other.  This might take a bit to work out the math.  Right now we pass in a `scale` parameter to `StaticImageLayer` that controls size relative to the image's lowest resolution.  So I guess I need to calculate that ahead of time to make it fit into the viewport...this should be doable.

Reply:
I just tried this and it seems to be working nicely:

```javascript
    this.width = viewWidth * overviewScale;
    this.height = this.width * (rasterSize.width / rasterSize.height);
```

Reply:
We could maybe add some logic to see which axis is longer, and then scale that axis (but just `scaleWidth` seems like a simple solution).